source/docs/software/advanced-controls/controls-glossary.rst

```diff
@@ -99,6 +153,12 @@ Controls Glossary
    system response
       The behavior of a :term:`system` over time for a given :term:`input`.
 
+   voltage
+      The measurement of how much an electric field is "pushing" electrons through a circuit. It is sometimes called "Electromotive Force", or "EMF". It is measured in units of "Volts". It always is defined between *two* points in a circuit. If one electron travels between two points that have one volt of EMF between them, it will have been accelerated to the point of having :math:`\frac{1}{6241509074000000000}` joules of energy.
+
+   viscous drag
+      The force generated from an object moving *relatively* slowly through non-turbulent fluid. It describes the most common type of "air resistance" an FRC robot would encounter, as well as losses in a gearbox from displacing grease. See `Drag (physics) <https://en.wikipedia.org/wiki/Drag_(physics)#Very_low_Reynolds_numbers:_Stokes'_drag>`__ on wikipedia for more info.
```

Comment:
"The force generated from an object moving *relatively* slowly through non-turbulent fluid, in which case the friction force is roughly proportional to the object's velocity (also called "linear drag" for this reason - faster-moving objects also begin to encounter "quadratic drag", which is proportional to the *square* of the velocity).

Reply:
Sounds good, adding an extra sentence to explain it. Somewhat purposefully leaving out the quadratic drag since (hopefully) robots aren't experiencing it.